flash/vision/backbones.py

```diff
@@ -47,7 +47,7 @@ def bolts_backbone_and_num_features(model_name: str) -> Tuple[nn.Module, int]:
     >>> bolts_backbone_and_num_features('simclr-imagenet')  # doctest: +ELLIPSIS
     (Sequential(...), 2048)
     >>> bolts_backbone_and_num_features('swav-imagenet')  # doctest: +ELLIPSIS
-    (Sequential(...), 3000)
+    (Sequential(...), 2048)
```

Comment:
this has been fixed already, right?

Reply:
Forgot to fix the doctest, in the last PR, for some reason tests still passed then but not now